english/time-travel/0/7-1.md

```diff
@@ -0,0 +1,7 @@
+**The whole world starts spinning 🌀 and you fall to the ground.**
```

Comment:
Please change it to :
**You feel dizzy from eating mushrooms. Your world starts spinning 🌀 and you fall to the ground.**

Reply:
And should add an option or jump into WIP.md

Reply:
Thanks!
What do you mean by WIP.md?

Reply:
Ok, sorry in this dungeon doesn't have `WIP.md`, so please add option to restart the game